### PR TITLE
[CDAP-17802] Handle 403 response when saving pipeline configuration

### PR DESCRIPTION
**Pull request by njbriggs**
Jira: https://cdap.atlassian.net/browse/CDAP-17802

### CODE REVIEW

**Comment by njbriggs** [comment]
Build: https://builds.cask.co/browse/CDAP-UDUT1028

File: cdap-ui/app/cdap/components/PipelineConfigurations/Store/ActionCreator.js [hunk 1]
```diff
@@ -233,12 +234,15 @@ const updatePipeline = () => {
...
   );
 
-  publishObservable.subscribe(() => {
-    PipelineDetailStore.dispatch({
-      type: PipelineDetailActions.SET_CONFIG,
-      payload: { config },
-    });
-  });
+  // Use pipe/map and not subscribe to not break error handling on other observers
+  publishObservable.pipe(
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/PipelineConfigurations/Store/ActionCreator.js:238]: if we do the following, would it still break other observers?

```
return pubishObservable.map((returnVal) => {
  PipelineDetailStore.dispatch({
    type: PipelineDetailActions.SET_CONFIG,
    payload: { config },
  });

  return returnVal;
});
```

Comment (njbriggs) [reply cdap-ui/app/cdap/components/PipelineConfigurations/Store/ActionCreator.js:238]: Good point. Let me check.

**Review by elfenheart — APPROVED** [review]

**Comment by njbriggs** [comment]
Squashed after approval.

**Pull request merged**